client/components/square.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+class SquareCream extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      pieceIsOn: false
+    };
+  }
```

Comment:
For now, since none of your components are updating their state, please remove your constructors and initial states. You can ignore this comment if you already have a working prototype that uses these states.